packages/react/src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -604,7 +604,8 @@ const SelectPanelSecondaryAction: React.FC<SelectPanelSecondaryActionProps> = ({
 
 const SelectPanelLoading = ({children = 'Fetching items...'}: React.PropsWithChildren) => {
   return (
-    <Status
+    <AriaStatus
+      announceOnShow
```

Comment:
Stupid question, are there styles attached to `<Status>` and is that different than `<AriaStatus>`

Reply:
@jonrohan no styles, just behavior for now 👀 They're both setup as a `Box` though internally 😅